bsd-user/riscv/target_arch_cpu.h

```diff
@@ -137,7 +137,7 @@ static inline void target_cpu_clone_regs(CPURISCVState *env, target_ulong newsp)
     }
 
     env->gpr[xA0] = 0; /* a0 */
-    env->gpr[5] = 0;   /* t0 */
+    env->gpr[xT0] = 0;   /* t0 */
```

Comment:
```suggestion
    env->gpr[xT0] = 0; /* t0 */
```

Reply:
Should I just click on 'sign off and commit suggestion' or send a new patch series with the changes?